python/dgl/distributed/partition.py

```diff
@@ -187,15 +287,39 @@ def partition_graph(g, graph_name, num_parts, out_path, num_hops=1, part_method=
     If node IDs and edge IDs are not shuffled to ensure that all nodes/edges in a partition
     fall into a contiguous ID range, DGL needs to store node/edge mappings (from
     node/edge IDs to partition IDs) in separate files (node_map.npy and edge_map.npy).
-    The node/edge mappings are stored in numpy files.
+    The node/edge mappings are stored in numpy files. **Note**: this format is
+    deprecated and will not be supported by the next release. In other words, the future
+    release will always shuffle node Ids and edge Ids when partitioning a graph.
+
+    If node Ids and edge Ids are shuffled, `node_map` and `edge_map` contains the information
```

Comment:
`Id` -> `ID` and `Ids` -> `IDs`.  Should be consistent.

Please scan other places as well.

Also, `If node IDs and edge IDs are shuffled` ->
```
If ``reshuffle = True``
```